noah/src/main/scala/edu/uci/ics/cloudberry/noah/news/WebhoseCollector.java

```diff
@@ -0,0 +1,201 @@
+package edu.uci.ics.cloudberry.noah.news;
+
+/**
+ * Collecting news articles from webhose.io and storing in JSON file.
+ *
+ * Created by Kaiyi Ma on 7/12/16.
+ */
+
+import com.buzzilla.webhose.client.WebhoseClient.WebhoseUrl;
+import com.buzzilla.webhose.client.WebhoseQuery;
+import com.buzzilla.webhose.client.WebhoseResponse;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.google.api.client.http.*;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.JsonObjectParser;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import org.apache.commons.io.FileUtils;
+import org.apache.http.NameValuePair;
+import org.apache.http.client.utils.URLEncodedUtils;
+import org.kohsuke.args4j.CmdLineException;
+import org.kohsuke.args4j.CmdLineParser;
+import org.kohsuke.args4j.Option;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.Properties;
+
+
+public class WebhoseCollector {
+    @Option(required = true,
+            name = "-a",
+            aliases = "--api",
+            usage = "Webhose API")
+    private String apiKey;
+
+    @Option(required = true,
+            name = "-o",
+            aliases = "--output",
+            usage = "output file directory")
+    private String filePath;
+
+    @Option(required = true,
+            name = "-t",
+            aliases = "--timestamp",
+            usage="timestamp for search")
+    private String ts;
+//    private static String propFileName = "";
+
+    public static void main( String[] args ) throws IOException {
+        new WebhoseCollector().doMain(args);
+    }
+
+    public void doMain(String[] args) throws IOException {
+        CmdLineParser parser = new CmdLineParser(this);
+        try{
+            parser.parseArgument(args);
+        } catch( CmdLineException e ) {
+            // if there's a problem in the command line,
```

Comment:
if the code is very clear about its functionality, we will not need this comment.